### ISSUE TITLE
Copying of cell output suppressed by user-select:none

### ISSUE THREAD

**Issue by verginer** [jupyter/jupyterlab#561]
In the classical notebook it was possible to copy cell output by selecting it with the cursor.
However this is no longer possible since the class of the output cell `p-Widget jp-Output-result jp-Output-executeResult`  prevents it specifically `user-select: none` and `cursor`.
```
.p-Widget {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
  cursor: default;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
```

**Comment by Dumbris** [comment]
I got the last sources from master repo right now and can't see this issue. 
Actually 
an .jp-Output-result.p-Widget selector override user-select: properties into "text". 
I have checked with last google-chrome and firefox 39. Works ok, I can copy text from output cell.

Only difference with Notebook is cursor property, in Lab it's pointer and in classical Notebook it's cursor: text
It can be fixed by adding "cursor: text" into .jp-Output-executeResult class. 
Can I do PR?
